frame/wormhole/issuing/ethereum/src/lib.rs

```diff
@@ -238,6 +239,7 @@ pub mod pallet {
 				register_info.decimals.as_u32() as u8,
 				backing_address,
 				register_info.token_address,
```

Comment:
Found it is called as source in smart contract?

In the token protocol context, we'd better to unify and using backing and issuing, because source and target are used for message layer.

Reply:
> > We have another problem when using this format of the token name.
> > In order to be compatible with some erc20 token names defined by bytes32, we restrict the name to 32 bytes.
> > If we add this `(BackingChain>IssuingChain)` to the end of the original name, it may become too long. @HackFisher
> 
> Since names are all normalized to the string name in mapping token contract:
> 
> https://github.com/darwinia-network/darwinia-bridge-sol/blob/bd3313a3f12b9118a2e1ae90bf58af83776e085d/contracts/mapping-token/contracts/darwinia/MappingERC20.sol#L21
> 
> Will the problem you mentioned still hold?

The contract can save this long type. And I'll remove this restrict https://github.com/darwinia-network/darwinia-common/blob/master/primitives/asset/src/token.rs#L35, this can be fixed.
But should we restrict the mapped times to prevent this name becomes long and long?